strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryPath.java

```diff
@@ -44,8 +46,12 @@ public Path getTarget()
     {
         return target;
     }
+    
+    public ArtifactEntry getArtifactEntry() {
+		return artifactEntry;
+	}
 
-    public RepositoryFileSystem getFileSystem()
+	public RepositoryFileSystem getFileSystem()
```

Comment:
Cosmetic: The indentation of this method seems incorrect.

Reply:
fixed